### PR TITLE
Added support for adding Watermark

### PR DESCRIPTION
**Pull request by venkinmobi**

### CODE REVIEW

**Comment by joshh-devrel** [comment]
@venkinmobi Can you update the changlog to include the below?
```
#### Next version
- Added watermark support for bidding banner, interstitial and rewarded ad formats.
```
Thanks!

File: ThirdPartyAdapters/inmobi/inmobi/src/main/java/com/google/ads/mediation/inmobi/renderers/InMobiBannerAd.java [hunk 1]
```diff
@@ -127,6 +129,11 @@ private void createAndLoadBannerAd(final Context context, final long placementId
 
     inMobiBannerWrapper.setListener(InMobiBannerAd.this);
 
+    String watermark = mediationBannerAdConfiguration.getWatermark();
+    if(watermark != null) {
+      inMobiBannerWrapper.setWatermarkData(new WatermarkData(watermark, WATERMARK_ALPHA));
```
Comment (joshuajplagonera) [inline ThirdPartyAdapters/inmobi/inmobi/src/main/java/com/google/ads/mediation/inmobi/renderers/InMobiBannerAd.java:134]: Is an empty (`""`) `watermark` data valid for InMobi? If not, maybe using `TextUtils.isEmpty(watermark) != null` is more applicable?

**Comment by joshh-devrel** [comment]
@venkinmobi Gentle bump, can you take a look at the comments? Thanks!

**Comment by joshh-devrel** [comment]
@venkinmobi Would you have an update on resolving the comments?

File: ThirdPartyAdapters/inmobi/inmobi/src/main/java/com/google/ads/mediation/inmobi/InMobiInterstitialWrapper.java [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 import androidx.annotation.VisibleForTesting;
 import com.inmobi.ads.InMobiInterstitial;
+import com.inmobi.ads.WatermarkData;
+
```
Comment (LTPhantom) [inline ThirdPartyAdapters/inmobi/inmobi/src/main/java/com/google/ads/mediation/inmobi/InMobiInterstitialWrapper.java:6]: nit: remove space

File: ThirdPartyAdapters/inmobi/inmobi/src/main/java/com/google/ads/mediation/inmobi/renderers/InMobiBannerAd.java (hunk 1 above)
Comment (LTPhantom) [reply ThirdPartyAdapters/inmobi/inmobi/src/main/java/com/google/ads/mediation/inmobi/renderers/InMobiBannerAd.java:134]: If using `TextUtils.isEmpty` as suggested, also apply for other formats.

**Comment by joshh-devrel** [comment]
@venkinmobi Gentle bump, can you resolve conflicts and the comments in the pull request?

**Review by LTPhantom — CHANGES_REQUESTED** [review]
Gentle bump to address the comments. Thanks.

**Comment by LTPhantom** [comment]
Closing this Pull Request since the watermark support has been already added and will be available in next adapter release.

**Pull request closed without merging**